.github/workflows/ci.yaml

```diff
@@ -218,6 +223,9 @@ jobs:
         run: |
           az extension add --name azure-iot   
           until `az iot edge deployment show-metric --deployment-id ${{ env.IOT_EDGE_DEPLOYMENT_ID }} --metric-id reportedSuccessfulCount --metric-type system --login '${{ secrets.IOTHUB_CONNECTION_STRING }}' | grep -q $DEVICE_ID`; do sleep 10 && echo wait; done
+          
+          # We add the 3 minutes sleep to let the container start up
```

Comment:
Nit: we might add to the comment a to-do item for improving this wait in the future